parquet/src/column/writer/mod.rs

```diff
@@ -703,6 +725,35 @@ impl<'a, E: ColumnValueEncoder> GenericColumnWriter<'a, E> {
             (Some(min), Some(max)) => {
                 update_min(&self.descr, &min, &mut self.column_metrics.min_column_value);
                 update_max(&self.descr, &max, &mut self.column_metrics.max_column_value);
+
+                // Check if min/max are still ascending/descending across pages
+                // Null pages aren't considered in this sort order
+                let null_page = (self.page_metrics.num_buffered_rows as u64)
+                    == self.page_metrics.num_page_nulls;
+                if !null_page {
+                    if let Some((latest_min, latest_max)) = &self.latest_non_null_data_page_min_max
```

Comment:
```suggestion
                    if let Some((last_min, last_max)) = &self.latest_non_null_data_page_min_max
```
I found this a little confusing whilst reviewing